### DISCUSSION TITLE
UTM 3.2.4 not available for ipad

### DISCUSSION THREAD

**Discussion by amith80kumar** [utmapp/UTM#6755] [UTM Help (iOS)] [answered]
Hello
I have installed Altstore on Ipad 12.5.7 and added the source https://alt.getutm.app/ however when I try to download UTM it only shows the UTM and UTM SE app and that requires IOS 14 or later. I do not see the UTM version 3.2.4 which is supported for my ipad. Can someone please explain how I can download the older version of UTM for my ipad running 12.5.7

Thank you

Regards
Amith

**Comment by osy** [comment, marked as answer]
Download the ipa https://github.com/utmapp/UTM/releases/download/v3.2.4/UTM.ipa

**Comment by amith80kumar** [comment]
Hello
I tried installing windows 10. It is complaining the below when it opens Windows boot manager. 
Status: 0xc0000017Info: There isnt enough memory available to create a ramdisk device.
Is there any help with this? I had an Ipad Air running 12.5.7.
Best RegardsAmith Kumar 

    On Thursday 17 October, 2024 at 09:54:22 am IST, osy ***@***.***> wrote:  
 
 


Download the ipa https://github.com/utmapp/UTM/releases/download/v3.2.4/UTM.ipa
